### PR TITLE
#657 Support updating account details for csv import

### PR DESCRIPTION
**Pull request by kaushikindianic**

### CODE REVIEW

File: app/config/routes.yml [hunk 1]
```diff
@@ -40,6 +40,10 @@ signup_thankyou:
     path: /signup/thankyou/{accountName}
     defaults:  { _controller: UserBase\Server\Controller\SignupController::signupThankYouAction }
 
+check_username:
+    path: /check/username
+    defaults: { _controller: UserBase\Server\Controller\SignupController::chkUserNameAction }
```
Comment (h-wang) [inline app/config/routes.yml]: @kaushikindianic plz use complete controller function name, e.g. checkUser...

File: src/Controller/SignupController.php [hunk 2]
```diff
@@ -238,12 +239,35 @@ public function signupThankYouAction(Application $app, Request $request, $accoun
...
 
         $data = array();
         return new Response($app['twig']->render(
             '@PreAuth/signup/thankyou.html.twig',
             $data
         ));
     }
+
+
+    public function chkUserNameAction(Application $app, Request $request)
```
Comment (h-wang) [inline src/Controller/SignupController.php]: @kaushikindianic complete function name

File: src/Controller/SignupController.php [hunk 3]
```diff
@@ -238,12 +239,35 @@ public function signupThankYouAction(Application $app, Request $request, $accoun
...
+    {
+        if ($request->isXmlHttpRequest()) {
+            $username =  $request->request->get('username');
+            $oUserRepo = $app->getUserRepository();
+
+            if ($oUserRepo->getByName($username)) {
+                return new JsonResponse(array(
```
Comment (h-wang) [inline src/Controller/SignupController.php]: @kaushikindianic @joostfaassen to me, normally a JsonResponse should be sent via an apiController

File: templates/admin/layout.html.twig [hunk 4]
```diff
@@ -12,8 +12,8 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
-    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="{{ path("bower") }}/bootstrap/dist/css/bootstrap.min.css">
```
Comment (h-wang) [inline templates/admin/layout.html.twig]: @kaushikindianic 👍

File: src/Controller/SignupController.php (hunk 3 above)
Comment (kaushikindianic) [inline src/Controller/SignupController.php]: @h-wang   Ajax request send result through  JasonResponse.

**Pull request merged**